packages/flutter_tools/lib/src/ios/migrations/remove_framework_link_and_embedding_migration.dart

```diff
@@ -2,26 +2,32 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'package:flutter_tools/src/base/common.dart';
```

Comment:
```dart
import '../../base/common.dart'
```

Reply:
Ugh Android Studio always does that without me noticing.